File: Data Structures & Algorithms/max-consecutive-ones/submission-0.go
func findMaxConsecutiveOnes(nums []int) int {
	count, max := 0, 0
	for _, n := range nums {
		if n == 1 {
			count++
		}else{
			count = 0
		}
		if count > max {
			max = count
		}
	}
	return max
}

File: Data Structures & Algorithms/sort-an-array/submission-0.py
class Solution:
    def sortArray(self, nums: List[int]) -> List[int]:
        count = defaultdict(int)
        minval,maxval = min(nums),max(nums)

        for num in nums:
            count[num]+=1
        index = 0
        for val in range(minval,maxval+1):
            while count[val]:
                nums[index] = val
                index +=1
                count[val] -=1

        return nums

        